StreamChatUITestsApp/AppDelegate.swift

```diff
@@ -0,0 +1,36 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        // Override point for customization after application launch.
+
+        disableAnimations()
+        setupUI()
+        return true
+    }
+
+    func setupUI() {
+        // Empty root view controller
+        let rootViewController = ViewController()
+
+        /// Embed in navigation controller
+        window = UIWindow()
+        window?.rootViewController = UINavigationController(rootViewController: rootViewController)
+        window?.makeKeyAndVisible()
+    }
+    
+    func disableAnimations() {
```

Comment:
Out of curiosity - why do we disable animations for the ui tests? I think they are relevant for the UI tests.

Reply:
When running UI tests you want to optimize the time it takes to run single UI tests. Disabling animations is one of the optimizations applied. 
At the moment we don't focus on the definition/performance of the animations. I believe we will have dedicated logic/test target for that particular logic.